java8/src/test/java/cucumber/runtime/java8/test/LambdaStepdefs.java

```diff
@@ -17,11 +19,14 @@ public LambdaStepdefs() {
             assertNotSame(this, lastInstance);
             lastInstance = this;
         });
-
         Given("^this data table:$", (DataTable peopleTable) -> {
             List<Person> people = peopleTable.asList(Person.class);
             assertEquals("Hellesøy", people.get(0).last);
         });
+        Integer alreadyHadThisManyCukes = 1;
+        Given("^I have 42 cukes in my belly$", () -> {
```

Comment:
Looks like your change breaks arguments. Try changing this line to the following to reproduce:

```java
Given("^I have (\\d+) cukes in my belly$", (Integer n) -> {
```

Any thoughts about how to deal with that?